lib/HTFeed/Stage/ImageRemediate.pm

```diff
@@ -369,7 +376,152 @@ sub _remediate_tiff {
       && $self->repair_tiff_exiftool( $infile, $outfile, $self->{newFields} );
 
     return $ret;
+}
+
+# Extract metadata from jpg/png and write to the corresponding tiff.
```

Comment:
New method. Delegates to the relevant `extract_$ext_metadata` writes extracted fields to output file.